pkg/modulewriter/tfwriter.go

```diff
@@ -205,8 +205,8 @@ func getProviders(bp config.Blueprint) []provider {
 	}
 
 	return []provider{
-		{"google", "hashicorp/google", "~> 4.84.0", gglConf},
-		{"google-beta", "hashicorp/google-beta", "~> 4.84.0", gglConf},
+		{"google", "hashicorp/google", ">= 4.84.0, <= 5.28.0", gglConf},
```

Comment:
I thought the plan was `~> 4.84.0, ~> 5.28.0`

Reply:
If we change this, it will require re-running PR-validation, but if that passes, it does not require the whole suite of tests to be re-run. That said, I don't think the difference in the change is huge.

Reply:
Nick and I discussed this offline. `~> 4.84.0, ~> 5.28.0` would not be able to find a version since both cannot be satisfied at once. So we decided to do the range.

Reply:
Playing around with it, I see the desired behavior best matches the choice here.